src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else if (days >= 3 && days < 7) {
+    return days * 40 - 20;
+  } else if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
Consider using constants for the daily rental cost, the discount thresholds, and the discount amounts to make the code more readable and maintainable, as per the checklist item 2. For example:

const DAILY_RENTAL_COST = 40;
const SHORT_TERM_THRESHOLD = 3;
const LONG_TERM_THRESHOLD = 7;
const SHORT_TERM_DISCOUNT = 20;
const LONG_TERM_DISCOUNT = 50;

This will help to avoid magic numbers in the code and make it easier to update the values in the future if needed.